changes/2024-TBD_key-store-admin/background.md

```diff
@@ -0,0 +1,224 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Key Store Administration Client & Configuration
+
+# Definitions
+
+## Conventions used in this document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+# Background
+
+The parameters needed to fetch Branch Keys for usage (encrypt, decrypt)
+are distinct from the parameters needed to administrate them
+(create, version/rotate, any future operations).
+
+The permissions needed for the persistence medium,
+which is DynamoDB by default,
+and to Key Management Service,
+which currently can only be AWS KMS,
+are also different between usage and administration activities.
+
+As such, it makes sense to create a
+new Branch Key Store Administration Client,
+that cannot facilitate usage,
+only administrative operations.
+
+## Alternatives to a new Client/Local Service?
+
+### Refactor Key Store
+
+Rather than creating a new Client/Local Service,
+Crypto Tools could refactor the existing
+Key Store Client to behave in the manner described below.
+
+This would be a breaking change,
```

Comment:
if we're adding new API operations, 
then it isn't necessarily a breaking change

Reply:
it seems like there are better reasons why it makes sense to have separate Admin vs. Cryptography/Dataplane/whatever, 
for the sake of allowing customers to more easily enable separation of permissions between the two.